weapons-manager.js

```diff
@@ -2205,7 +2206,9 @@ const _updateMenu = () => {
     objectMenuEl.classList.toggle('open', true);
     itemLabel.classList.toggle('open', true);
     itemIcon.classList.toggle('open', true);
-
+    if (document.monetization.state == 'started') {
+      itemMonetizedIcon.classList.toggle('open', true);

```

Comment:
We'll probably need to make this per-item once items are hooked in.

Reply:
Yeah. Right now it is in-line with the current implementation, which is -- if monetization is enabled, then monetization is rotated between the items.. to the owner's monetization pointer.